### PR TITLE
(FACT-1997) Collect statistics using Beaker

### PR DESCRIPTION
**Pull request by oanatmaria**

### CODE REVIEW

File: acceptance/presuite/CollectStatistics.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+require 'pathname'
+test_name "Collect facter statistics" do
+    @log_dir = File.join(Pathname.new(File.expand_path('..', __dir__)), "log_dir")
```
Comment (gimmyxd) [inline acceptance/presuite/CollectStatistics.rb]: I don't think you need instance variables here

File: acceptance/presuite/CollectStatistics.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
+require 'pathname'
```
Comment (gimmyxd) [inline acceptance/presuite/CollectStatistics.rb]: the file name should be snake_case, as it is a ruby convention

File: acceptance/Rakefile [hunk 3]
```diff
@@ -0,0 +1,66 @@
+require 'beaker-puppet'
```
Comment (gimmyxd) [inline acceptance/Rakefile]: Most likely you don't need this rakefile at all, only if you need to define specific rake tasks for statistax

File: acceptance/config.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+[
```
Comment (gimmyxd) [inline acceptance/config.json]: is this file needed?

File: acceptance/config/aio/options.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (gimmyxd) [inline acceptance/config/aio/options.rb]: You only need the `type: 'aio'` here

File: acceptance/config.json (hunk 4 above)
Comment (oanatmaria) [reply acceptance/config.json]: Yes, because in the test I copy this file to every agent in order to be able to run statistax tool

File: acceptance/presuite/collect_statistics.rb [hunk 6]
```diff
@@ -0,0 +1,67 @@
+require 'pathname'
```
Comment (gimmyxd) [inline acceptance/presuite/collect_statistics.rb]: would it make sense to break this in multiple files and make it a bit more configurable?
like:
 - install/uninstall puppet-agent
 - install facter-statistax
 - install/unistall facter gem(specific version)
 - run the actuall metrics, which could be reused on the same machines?

or this is intended for single  use, single gem version?
